@xen-orchestra/lite/src/components/Tree.tsx

```diff
@@ -73,21 +73,26 @@ interface Computed {}
 // Inspired by https://mui.com/components/tree-view/#contentcomponent-prop.
 const CustomContent = React.forwardRef(function CustomContent(props: CustomContentProps, ref) {
   const { classes, className, label, expansionIcon, nodeId, to } = props
-  const { handleExpansion, handleSelection, selected } = useTreeItem(nodeId)
+  const { focused, handleExpansion, handleSelection, selected } = useTreeItem(nodeId)
   const history = useHistory()
 
+  React.useEffect(() => {
```

Comment:
Can you use reaclette?

Reply:
It's seems difficult to use raclette with `forwardRef`